### PR TITLE
[Notebook] new notebook entry causes console error #3440

### PR DESCRIPTION
**Pull request by nikhilmandlik**
### Resolves: 
#3440

### Author Checklist:
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | N/A
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y

### CODE REVIEW

File: src/plugins/notebook/components/Notebook.vue [hunk 1]
```diff
@@ -431,7 +431,9 @@ export default {
         },
         async updateDefaultNotebook(notebookStorage) {
             const defaultNotebookObject = await this.getDefaultNotebookObject();
-            if (defaultNotebookObject.identifier.key !== notebookStorage.notebookMeta.identifier.key) {
+            if (!defaultNotebookObject) {
+                setDefaultNotebook(this.openmct, notebookStorage);
+            } else if (defaultNotebookObject.identifier.key !== notebookStorage.notebookMeta.identifier.key) {
```
Comment (deeptailor) [inline src/plugins/notebook/components/Notebook.vue:436]: We should always be checking keystrings. Not including the namespace property will cause issues down the road

**Review by deeptailor — CHANGES_REQUESTED** [review]
One comment regarding keys

File: src/plugins/notebook/components/Notebook.vue (hunk 1 above)
Comment (nikhilmandlik) [reply src/plugins/notebook/components/Notebook.vue:436]: or is it better to use, `objectEquals` from `object-utils.js`?

Comment (nikhilmandlik) [reply src/plugins/notebook/components/Notebook.vue]: using `makeKeyString `

**Review by deeptailor — APPROVED** [review]
LGTM

**Comment by deeptailor** [comment]
## Reviewer Checklist
Changes appear to address issue? YES
Appropriate unit tests included? N/A
Code style and in-line documentation are appropriate? YES
Commit messages meet standards? YES
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) YES

**Pull request merged**